iron/distribution.yaml

```diff
@@ -6225,6 +6225,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_topic.git
       version: iron
     status: maintained
+  rrt_planner:
+    doc:
+      type: git
+      url: https://github.com/daviddorf2023/rrt_planner.git
```

Comment:
One problem I have with this is that the name `rrt_planner` is too generic.  I can imagine there being multiple implementations of an RRT planner in ROS.  This comes under our naming policies in https://www.ros.org/reps/rep-0144.html#global-guidelines .

What I'll suggest here is that this package gets renamed to something like `daviddorf_rrt_planner`, which I think would satisfy REP-144 above.